contrib/connector/fc/linuxfc.go

```diff
@@ -176,3 +176,28 @@ func trimDoubleQuotesInText(str string) string {
 	}
 	return str
 }
+
+func getMultipathDevice(deviceWWN string) (string, error) {
+	cmd := fmt.Sprintf("ls -l /dev/disk/by-id/ | grep %s", deviceWWN)
+	out, err := connector.ExecCmd("/bin/bash", "-c", cmd)
+	if err != nil {
+		errMsg := fmt.Sprintf("Error occurred when find DM of wwn %s: %s, %v", deviceWWN, out, err)
```

Comment:
Error strings should not be capitalized.
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Same as other error strings